src/current/molt/molt-fetch.md

```diff
@@ -945,6 +1049,65 @@ molt fetch \
+{{site.data.alerts.end}}
+
+The following `molt fetch` command uses [`failback` mode](#fail-back-to-source-database) to securely replicate changes from CockroachDB back to a MySQL database. This assumes that you migrated data from MySQL to CockroachDB, and want to keep the data consistent on MySQL in case you need to roll back the migration.
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+molt fetch \
+--source 'postgres://root@localhost:26257/defaultdb?sslmode=verify-full' \
+--target 'mysql://root:password@localhost/molt?sslcert=.%2fsource_certs%2fclient.root.crt&sslkey=.%2fsource_certs%2fclient.root.key&sslmode=verify-full&sslrootcert=.%2fsource_certs%2fca.crt' \
+--table-filter 'employees, payments' \
+--non-interactive \
+--logging debug \
+--replicator-flags "--tlsCertificate ./certs/server.crt --tlsPrivateKey ./certs/server.key" \
+--mode failback \
+--changefeeds-path 'changefeed-secure.json'
+~~~
+
+- `--source` specifies the connection string of the CockroachDB database to which you migrated.
+- `--target` specifies the connection string of the MySQL database acting as the failback target.
+- `--table-filter` specifies that the `employees` and `payments` tables should be watched for change events.
+- `--replicator-flags` specifies the paths to the server certificate (`--tlsCertificate`) and key (`--tlsPrivateKey`) that correspond to the client certs defined by `sink_query_parameters` in the changefeed override JSON.
+- `--changefeeds-path` specifies the path to `changefeed-secure.json`, which contains the following setting override:
+
+	{% include_cached copy-clipboard.html %}
+	~~~ json
+	{
+	    "sink_query_parameters": "client_cert={base64 cert}&client_key={base64 key}&ca_cert={base64 CA cert}"
+	}
+	~~~
+
+	`client_cert`, `client_key`, and `ca_cert` are [webhook sink parameters]({% link {{ site.current_cloud_version }}/changefeed-sinks.md %}#webhook-parameters) that must be base64- and URL-encoded (for example, `base64 -i ./client.crt | jq -R -r '@uri'`).
+
+	{{site.data.alerts.callout_success}}
+	For details on the default changefeed settings and how to override them, see [Changefeed override settings](#changefeed-override-settings).
+	{{site.data.alerts.end}}
+
+The preceding `molt fetch` command issues the equivalent [`CREATE CHANGEFEED`]({% link {{ site.current_cloud_version }}/create-changefeed.md %}) command, using the default and explicitly overriden changefeed settings:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE CHANGEFEED FOR TABLE employees, payments 
+  INTO 'webhook-https://localhost:30004/defaultdb/public?client_cert={base64 cert}&client_key={base64 key}&ca_cert={base64 CA cert}' 
+  WITH updated, resolved = '1s', min_checkpoint_frequency = '1s', initial_scan = 'no', cursor = '2024-09-11T16:33:35Z', webhook_sink_config = '{\"Flush\":{\"Bytes\":1048576,\"Frequency\":\"1s\"}}'
+~~~
+
+This results in initial output like the following:
```

Comment:
This may be redundant, but for clarity:
```suggestion
Running the preceding `molt fetch` command or the equivalent `CREATE CHANGEFEED` command results in initial output like the following:
```

Reply:
I don't want to suggest manually running that changefeed command, so I slightly reworded.